react/FormConfirmationCheckbox.tsx

```diff
@@ -57,19 +59,21 @@ function FormConfirmationCheckbox(props: Props) {
           <IOMessage
             id={checkboxLabel}
             values={{
-              firstLink: firstLabelLink && (
-                <LabelLink
-                  text={firstLabelLink.text}
-                  url={firstLabelLink.url}
-                  ordinalPosition="first"
-                />
+              firstLink: (
```

Comment:
Isn't that kind of a different behavior? Before, there would be no `<a>` tag if `firstLabelLink` wasn't provided. I think this may cause layout differences if `labelLink` is being used while relying on this condition.

Reply:
Nice catch! But in this case, this wouldn't be a problem. 

Notice that `firstLink` is being passed to `<IOMessage />` as a variable (via `values`), which means that this `<a>` will be interpolated in a string such as "I accept the {firstLink}.". And notice how it **needs** `firstLabelLink` to **not** be `undefined`, right? Because it relies on the values passed to that object. It doesn't make sense for a user to relly on a scenario where they are using `{firstLink}` in their ICU string and not passing a value to `firstLabelLink`.

Oh! And if `firstLabelLink` is not provided, and the user is **not** using `{firstLink}` in their ICU message, then it makes no difference :)

This is also valid for `secondLink` below 👇

Reply:
Got it!